upgrade.php

```diff
@@ -1623,7 +1622,39 @@ function upgrade_1824_sqlite() {
           {PRIMARY} (`on_vacation`,`notified`),
           CONSTRAINT `vacation_notification_pkey` FOREIGN KEY (`on_vacation`) REFERENCES `vacation` (`email`) ON DELETE CASCADE);
     ");
+}
+
+function upgrade_1825_mysql_pgsql() {
+    # Add support for multiple servers
+    $server = table_by_key('server');
+    db_query_parsed("
+        CREATE TABLE {IF_NOT_EXISTS} $server (
+          server VARCHAR(100) {LATIN1} NOT NULL,
+          description VARCHAR(255) {UTF-8} NOT NULL,
+          address VARCHAR(100) {LATIN1} NOT NULL,
+          created {DATE},
+          modified {DATECURRENT},
+          PRIMARY KEY (server)
+        ) {MYISAM} ;
+    ");
+}
+
+function upgrade_1825_sqlite() {
+    $server = table_by_key('server');
+    db_query_parsed("
+        CREATE TABLE $server (
+         `server` varchar(100) NOT NULL,
+         `description` varchar(255) NOT NULL,
+         `address` varchar(100) NOT NULL,
+         `created` {DATE},
+         `modified` {DATECURRENT},
+         {PRIMARY} (`server`));
+    ");
+}
 
+function upgrade_1826() {
+    $domain = table_by_key('domain');
+    _db_add_field($domain, 'primarymx','VARCHAR(100) {LATIN1} NOT NULL','transport');
```

Comment:
_db_add_field calls table_by_key() itsself, which means we run into double-mapping of the name. 
Just call _db_add_field('domain', ...).